src/test/client/ops.cc

```diff
@@ -0,0 +1,48 @@
+// -*- mode:C++; tab-width:8; c-basic-offset:2; indent-tabs-mode:t -*-
+// vim: ts=8 sw=2 smarttab
+/*
+ * Ceph - scalable distributed file system
+ *
+ * Copyright (C) 2022 Red Hat
+ *
+ * This is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU Lesser General Public
+ * License version 2.1, as published by the Free Software
+ * Foundation.  See file COPYING.
+ *
+ */
+
+#include <iostream>
+#include <errno.h>
+#include "TestClient.h"
+#include "gmock/gmock.h"
+#include "gtest/gtest.h"
+#include "gtest/gtest-spi.h"
+#include "gmock/gmock-matchers.h"
+#include "gmock/gmock-more-matchers.h"
+
+using ::testing::AnyOf;
+using ::testing::Gt;
+using ::testing::Eq;
+
+TEST_F(TestClient, CheckDummyOP) {
+  int res = client->check_dummy_op(myperm);
+  ASSERT_EQ(res, -EOPNOTSUPP);
+}
+
+TEST_F(TestClient, CheckUnknownSessionOp) {
+  int res = client->check_unknown_session_op();
+  ASSERT_EQ(res, -EINVAL);
```

Comment:
The `res` here is the result of `session->con->send_message2(std::move(msg));`, no matter what the `op` is it will always succeed and `res` is 0.

Reply:
Yeah, as per our discussion, I made the changes accordingly and now we only declare success if the client is actually blocklisted